blog/app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
Lets see haml-rails gem be properly integrated. It that doesn't work, discuss amongst all and come up with the solution.